sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -798,6 +799,13 @@ class SdkSynchronizer private constructor(
             account
         )
 
+    override suspend fun getCustomUnifiedAddress(account: Account, request: UnifiedAddressRequest): String =
```

Comment:
A corresponding change to `Synchronizer.kt` is necessary so that this actually overrides something. See `getUnifiedAddress` in that file.